packages/forma-36-react-components/src/components/Button/Button.stories.tsx

```diff
@@ -5,7 +5,7 @@ import { text, boolean, select } from '@storybook/addon-knobs';
 import { iconName } from '../Icon/constants';
 
 import Button from './Button';
-import notes from './Button.md';
+import notes from './README.md';
```

Comment:
I'd recommend keeping this to match the component name for better developer experience when editing multiple readmes at the same time

Reply:
@m10l sure, I thought that it would be useful to have it displayed in here https://github.com/contentful/forma-36/tree/readme-button-proposal/packages/forma-36-react-components/src/components/Button. If it's a Button.md it won’t be displayed, I think

Reply:
@burakukula I also like to have the README displayed in github
I vote for keeping it README